src/main/java/com/iota/iri/service/transactionpruning/jobs/MilestonePrunerJob.java

```diff
@@ -125,6 +127,7 @@ public void process() throws TransactionPruningException {
                     }
 
                     getTransactionPruner().saveState();
+                    getSnapshot().setIndex(getCurrentIndex());
```

Comment:
This incorrect,
The snapshot currentIndex (the milestone where we took the snapshot) shouldn't be the last pruned index...

Remember snapshotting and pruning are two separate things

Reply:
Oh good catch, i thought i deleted that...